source/nodejs/adaptivecards-designer/src/dialog.ts

```diff
@@ -121,7 +139,12 @@ export abstract class Dialog {
             this._overlayElement.appendChild(dialogFrameElement);
 
             document.body.appendChild(this._overlayElement);
-            dialogFrameElement.focus();
+
+            let focusableElements = dialogFrameElement.querySelectorAll('a[href], area[href], input:not([disabled]), select:not([disabled]), textarea:not([disabled]), button:not([disabled]), [tabindex="0"]');
```

Comment:
Probably a corner case, but what about elements that have tabindex > 0?

Reply:
`>0` tabindex is broadly considered a [bad](https://karlgroves.com/2018/11/13/why-using-tabindex-values-greater-than-0-is-bad) [practice](https://web.dev/tabindex/) for [accessibility](https://dequeuniversity.com/rules/axe/4.0/tabindex) [reasons](https://adrianroselli.com/2014/11/dont-use-tabindex-greater-than-0.html)... I think we'd really want to discourage its use.